src/components/AdminPage/AdminPanel/AdminPanel.js

```diff
@@ -1,5 +1,123 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import styles from "./AdminPanel.module.css";
+import { auth } from "../../../firebase";
+import { signInWithEmailAndPassword, signOut } from "firebase/auth";
+import {
+  Button,
+  Form,
+  FormGroup,
+  FormLabel,
+  FormControl,
+} from "react-bootstrap";
```

Comment:
Remove unused variables, unused imports, etc. See what eslint displays in the console.